example/config.yml

```diff
@@ -6,13 +6,44 @@ media_folder: "assets/uploads"
 
 collections: # A list of collections the CMS should be able to edit
   - name: "posts" # Used in routes, ie.: /admin/collections/:slug/edit
-    label: "Post" # Used in the UI, ie.: "New Post"
+    label: "(EN) Post" # Used in the UI, ie.: "New Post"
```

Comment:
Nitpick: drop the "(EN) " prefix - if this update is too in-your-face it could cause confusion. We want it to be clear that this is optional.